packages/popover/README.md

```diff
@@ -60,16 +60,20 @@ The popover component will be automatically positioned relative to its nearest p
 ## Properties
 
 | Prop               | Type                                                                                           | Description                                                                                                                                                                                                              | Default    |
-| ------------------ | ---------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------- |
+| ------------------ | ---------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------- | --- |
```

Comment:
did we accidently add an extra column here?

Reply:
Yep, fixed 👍